docs/guide/snaps-development-guide.md

```diff
@@ -317,12 +314,18 @@ In the future, MetaMask will create some way for users to more easily discover s
 
 ## Resources and Tools
 
-You can review the growing number of [example snaps](https://github.com/MetaMask/snaps-skunkworks/tree/main/packages/examples) maintained by MetaMask, and you may wish to check out the first functional key management snap ever made, [the Filecoin Snap, or `filsnap`](https://github.com/Chainsafe/filsnap).
-
-MetaMask also maintains a growing list of tools to help developers debug, build and maintain Snaps:
+MetaMask maintains a growing list of tools to help developers debug, build and maintain Snaps:
 
 - [Snaps Inspector](https://inspector.open-rpc.org/?request[jsonrpc]=2.0&request[method]=confirm&request[params][0]=hello&request[params][1]=more&request[params][2]=lorem%20ipsum&request[id]=0&url=npm:@metamask/test-snap-confirm&customTransport[type]=plugin&customTransport[name]=Snaps&customTransport[transport][type]=postmessageiframe&customTransport[transport][name]=PostMessageIframe&customTransport[uri]=https://xops.github.io/inspector-snaps-transport/) - An API tool to make JSON-RPC requests directly to Snaps.
 - [Snaps OpenRPC Generator](https://github.com/xops/snaps-openrpc-generator) - A project skeleton generator that creates a Snap and documentation from your [OpenRPC document](https://spec.open-rpc.org/#openrpc-document)
-- [5-Minute Snap Tutorial](https://github.com/Montoya/gas-fee-snap#readme)
+- Snaps Tutorials: 
+  - A 5-minute tutorial that uses the `network-access` permission: [Gas Fee Snap Tutorial](https://github.com/Montoya/gas-fee-snap#readme)
+  - A 30-minute tutorial that uses the `manageState` permission: [Address Book Snap Tutorial](https://github.com/Montoya/address-book-snap-tutorial#readme)
+
+You can also try the following example snaps. Each one is fully functional and open-source:
+
+  - [Filsnap for Filecoin support](https://github.com/Chainsafe/filsnap/)
+  - [BitcoinSnap for Bitcoin support](https://github.com/KeystoneHQ/btcsnap)
+  - [Password Manager Snap](https://github.com/ritave/snap-passwordManager)
 
-Finally, if you need help, you can ask for help in the [MetaMask/snaps-skunkworks](https://github.com/MetaMask/snaps-skunkworks) repository.
+Finally, if you need help, you can ask for help in our [discussion board](https://github.com/MetaMask/snaps-skunkworks/discussions). 
```

Comment:
We should encourage to open issues as well in the repo if they encounter errors!

Reply:
OK added in latest commit.